### PR TITLE
WEB-9678 VT blog index styles and fixes

### PR DESCRIPTION
**Pull request by Wmiranda1**

### CODE REVIEW

File: components/layouts/blog/blog-homepage-section.hbs [hunk 1]
```diff
@@ -163,6 +163,9 @@
             </div><!-- .content-slider__cards-list .grid-col-content__four--homepage-blog -->
 
         </div><!-- .content-slider-blog-->
+        <div class="content-block--homepage-blog--blog-button">
```
Comment (fienen) [inline components/layouts/blog/blog-homepage-section.hbs:166]: This has two modifiers in the class name.

File: components/layouts/blog/blog-homepage-section.hbs [hunk 2]
```diff
@@ -163,6 +163,9 @@
...
+            <a href="/blog/" class="button--medium-primary">View All</a>
```
Comment (fienen) [inline components/layouts/blog/blog-homepage-section.hbs:167]: Double check this. I think it should be button--large-primary

File: components/layouts/blog/blog-post.hbs [hunk 3]
```diff
@@ -22,16 +22,10 @@
...
-                <a href="https://twitter.com/Vitamin T" title="Twitter" target="_blank" class="social-icon__link">
-                    <img class="social-icon--twitter" /><span class="show-for-sr">Twitter</span>
-                </a>
-                <a href="https://www.facebook.com/aquent/" title="Facebook" target="_blank" class="social-icon__link">
-                    <img class="social-icon--facebook" /><span class="show-for-sr">Facebook</span>
-                </a>
+                 <a href="https://www.linkedin.com/company/aquent" title="LinkedIn" target="_blank"
+                    class="social-icon__link--linkedin"></a>
+                <a href="https://twitter.com/Vitamin T" title="Twitter" target="_blank" class="social-icon__link--twitter"></a>
+                <a href="https://www.facebook.com/aquent/" title="Facebook" target="_blank" class="social-icon__link--facebook"></a>
```
Comment (fienen) [inline components/layouts/blog/blog-post.hbs:28]: These shouldn't be links to our profiles, they should be pre-formatted links for sharing a post. Look at the AQ AU blog as for the link formats.

**Review by fienen — CHANGES_REQUESTED** [review]
A few notes inline that need checked, tweaked.

File: components/layouts/blog/blog-homepage-section.hbs (hunk 1 above)
Comment (Wmiranda1) [reply components/layouts/blog/blog-homepage-section.hbs]: Fixed

File: components/layouts/blog/blog-homepage-section.hbs (hunk 2 above)
Comment (Wmiranda1) [reply components/layouts/blog/blog-homepage-section.hbs]: Correct and fixed.

File: components/layouts/blog/blog-post.hbs (hunk 3 above)
Comment (Wmiranda1) [reply components/layouts/blog/blog-post.hbs]: Updated link format.

**Pull request merged**